common/src/main/resources/assets/ftbquests/lang/fr_fr.json

```diff
@@ -0,0 +1,482 @@
+	"ftbquests.dependency": "Dépendance",
+	"ftbquests.null": "Aucun",
+	"ftbquests.disable_completion_toast": "Désactivé la bulle d'achèvement",
+	"ftbquests.file": "Pack de mods",
+	"ftbquests.file.completed": "Tous les chpitres sont complétés !",
+	"ftbquests.file.allow_take_quest_blocks": "Autoriser à prendre les livres de quêtes",
+	"ftbquests.file.allow_take_quest_blocks.tooltip": "Autoriser l'obtention de blocs de quête à partir de l'interface de quêtes.",
+	"ftbquests.file.emergency_items": "Objets d'urgence",
+	"ftbquests.file.emergency_items_cooldown": "Temps d'attente entre les objets d'urgence",
+	"ftbquests.file.emergency_items_cooldown.tooltip": "Temps d'attente pour la réapparition des objets d'urgence.",
+	"ftbquests.file.defaults": "Valeurs par défaut",
+	"ftbquests.file.defaults.reward_team": "Récompense 'Récomense d'équipe",
+	"ftbquests.file.defaults.consume_items": "Tâche 'Consommer les objets'",
+	"ftbquests.file.defaults.autoclaim_rewards": "Récompense 'Récupération automatique'",
+	"ftbquests.file.defaults.quest_shape": "Forme de quête",
+	"ftbquests.file.defaults.quest_disable_jei": "Quête 'Désactiver les recettes JEI'",
+	"ftbquests.file.drop_loot_crates": "Entités donne des caisses de butin",
+	"ftbquests.file.loot_crate_no_drop": "Entités donne des caisses de butin vides",
+	"ftbquests.file.disable_gui": "Désactiver l'interface",
+	"ftbquests.file.full_screen_quest_view": "Vue des quêtes en plein écran",
+	"ftbquests.file.grid_scale": "Taille de la grille",
+	"ftbquests.file.lock_message": "Bloquer les messages",
+	"ftbquests.file.progression_mode": "Mode de progression",
+	"ftbquests.file.detection_delay": "Intervale miinmum pour l'auto détection des objets",
+	"ftbquests.file.detection_delay.tooltip": "Intervale minimum (en ticks) entre les vérifications de l'inventaire d'un joueur quand il a changé\nLes plus grandes valeurs sont préconisées pour les performances des serveurs, surtout si l'inventaire des joueurs change souvent, cela veut aussi dire un temps plus long pour les messages d'achèvement automatique des quêtes.",
+	"ftbquests.chapter.progression_mode": "Mode de progression",
+	"ftbquests.file.progression_mode.tooltip": "Linéaire: La quête doit être complétée avant que les quêtes dépendantes ne soient commencées\nFlexible: Les tâches des quêtes dépendantes peuvent être avancées à tout moment,\n  mais les quêtes dépendantes ne peuvent pas être complétée avant que la dépendance ne le soit",
+	"ftbquests.file.progression_mode.default": "Par Défaut",
+	"ftbquests.file.progression_mode.linear": "Linéaire",
+	"ftbquests.file.progression_mode.flexible": "Flexible",
+	"ftbquests.loot.entitydrops": "Butin des entités",
+	"ftbquests.loot.entitytype.passive": "Passif",
+	"ftbquests.loot.entitytype.monster": "Monstre",
+	"ftbquests.loot.entitytype.boss": "Boss",
+	"ftbquests.chapter": "Chapitre",
+	"ftbquests.chapters": "Chapitres",
+	"ftbquests.chapter.completed": "Chapitre complété!",

```

Comment:
In French there is a space between the word "complété" and the punctuation mark "!"